### PR TITLE
docs: Auto-generating e-Way Bill JSON from Sales Invoice

### PR DESCRIPTION
**Pull request by sagarvora**

### CODE REVIEW

File: erpnext_com/www/docs/user/manual/en/regional/india/auto-generate-e-way-bill-JSON.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+<!-- add-breadcrumbs -->
+# Auto-generating e-Way Bill JSON from Sales Invoice
+
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/regional/india/auto-generate-e-way-bill-JSON.md]: explain the concept

File: erpnext_com/www/docs/user/manual/en/regional/india/auto-generate-e-way-bill-JSON.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+### Prerequisites
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/regional/india/auto-generate-e-way-bill-JSON.md]: h2 after h1, our CSS was updated

File: erpnext_com/www/docs/user/manual/en/regional/india/auto-generate-e-way-bill-JSON.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+- GST Accounts must be set in the GST Settings DocType.
+
+- Fields required to generate a valid e-Way Bill must be entered in Sales Invoice. To identify the required fields and for more information, you can refer to the <a href="https://docs.ewaybillgst.gov.in/html/formatdownloadnew.html">JSON schema</a>.
+
+### Instructions
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/regional/india/auto-generate-e-way-bill-JSON.md]: h2

File: erpnext_com/www/docs/user/manual/en/regional/india/auto-generate-e-way-bill-JSON.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
+<!-- add-breadcrumbs -->
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/regional/india/auto-generate-e-way-bill-JSON.md]: add file in the index

File: erpnext_com/www/docs/user/manual/en/regional/india/auto-generate-e-way-bill-JSON.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+- Fields required to generate a valid e-Way Bill must be entered in Sales Invoice. To identify the required fields and for more information, you can refer to the <a href="https://docs.ewaybillgst.gov.in/html/formatdownloadnew.html">JSON schema</a>.
+
+### Instructions
+
+1. After entering the required data, submit the Sales Invoice(s).
+
+1. You should now see a button at the top-right corner labelled **Generate e-Way Bill JSON**.
+
+	<img class="screenshot" src="/docs/assets/img/regional/india/ewb_invoice_button.png">
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/regional/india/auto-generate-e-way-bill-JSON.md]: update all screenshots to also show the navigation on the top left

File: erpnext_com/www/docs/user/manual/en/regional/india/auto-generate-e-way-bill-JSON.md [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+
+1. If you want to generate a JSON file for multiple invoices, you can select the relevant invoices from Sales Invoice List and find the **Generate e-Way Bill JSON** button under the **Actions** menu in the top-right corner.
+
+	<img class="screenshot" src="/docs/assets/img/regional/india/ewb_list_button.png">
+
+1. If all goes well, clicking this button will download the auto-generated JSON file into your device.
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/regional/india/auto-generate-e-way-bill-JSON.md]: change wording, why would it "not go well"?

File: erpnext_com/www/docs/user/manual/en/regional/india/auto-generate-e-way-bill-JSON.md [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+
+1. Log in to the <a href="https://ewaybillgst.gov.in">GST e-Way Bill System</a> using your credentials.
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/regional/india/auto-generate-e-way-bill-JSON.md]: too many steps. Add a separate sub heading here

**Comment by sagarvora** [comment]
@prasadarr I will do it over the weekend and update the PR accordingly.

**Comment by prasadarr** [comment]
@sagarvora I'm waiting for the changes

**Comment by sagarvora** [comment]
#380

**Pull request closed without merging**